Makefile

```diff
@@ -212,6 +212,12 @@ test-relay-integration:
 	pytest tests/relay_integration -vv
 	@echo ""
 
+test-api-docs:
```

Comment:
This won't be run in CI. You need a `TEST_SUITE=api-docs` with both python and node dependencies, and a `travis-test-api-docs` make target.

Reply:
Which is, needless to say, going to be a lot of extra overhead.

I would strongly prefer combining with the Plugins test suite, which isn't a bottleneck job and has python+node dependencies.

Reply:
Maybe like, renaming to `test-other` and removing TEST_GROUP. cc @wedamija

Reply:
@joshuarli so we aren't adding this to CI yet, and it's going to be run in it's own GH action, not travis

cc @billyvg

Reply:
Oh, okay. My comments on combining w/ plugins still applies though.

Reply:
@joshuarli Are you talking about `TOTAL_TEST_GROUPS`? Why would we remove that?

Reply:
No, the `ifndef TEST_GROUP` in `test-plugins`. It's not used to shard plugins tests and if things were combined, there wouldn't be a clear way to shard I don't think. It's not a bottleneck job anyways.

Reply:
@joshuarli getsentry uses it

Reply:
I've rebased onto master so there are no longer `Makefile` changes.